### PR TITLE
pythonPackages.tensorflow 1.0

### PR DESCRIPTION
**Pull request by edwtjo**
###### Motivation for this change

Updates tensorflow to 1.0/python3 and adds protobufs 3.2

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -20162,6 +20161,22 @@ in {
     };
   };
 
+  pyext = buildPythonPackage rec {
+    name = "pyext-${version}";
+    version = "0.7";
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/b0/be/9b6005ac644aaef022527ce49617263379e49dbdbd433d1d3dd66d71f570/pyext-0.7.tar.gz";
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: use `fetchPypi`

Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: and can you put this new package in a new file in `pkgs/development/python-modules/pyext/default.nix`

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/tensorflow/default.nix [hunk 2]
```diff
@@ -17,20 +17,20 @@
...
   name = "${pname}-${version}";
   format = "wheel";
 
   src = fetchurl {
     url = if stdenv.isDarwin then
-      "https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-${version}-py2-none-any.whl" else
-      "https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-${version}-cp27-none-linux_x86_64.whl";
+      "https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-${version}-py3-none-any.whl" else
+      "https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-${version}-cp35-cp35m-linux_x86_64.whl";
     sha256 = if stdenv.isDarwin then
```
Comment (jyp) [inline pkgs/development/python-modules/tensorflow/default.nix]: Could you please leave the 2.7 version in? Like this: https://github.com/NixOS/nixpkgs/pull/20794/commits/69c9298259c21ef52c41b3e6f9f1b760ffa8ee41

**Comment by jyp** [comment]
I think it would be best to keep the support for python 2.7. This old PR shows how to do it: https://github.com/NixOS/nixpkgs/pull/20794/commits

**Comment by jyp** [comment]
Look very nice thanks!

**Comment by edwtjo** [comment]
Cherry picked to master

**Pull request closed without merging**